PhpAmqpLib/Wire/AMQPWriter.php

```diff
@@ -180,28 +180,32 @@ public function write_signed_octet($n)
     /**
      * Write an integer as an unsigned 16-bit value
      *
-     * @param $n
+     * @param int $shortValue
      * @return $this
      * @throws \PhpAmqpLib\Exception\AMQPInvalidArgumentException
      */
-    public function write_short($n)
+    public function write_short($shortValue)
```

Comment:
Ha, no worries.  Very valid and good point.  Thank you...

The change is made and rebased btw.